unofficial_irods_documentation/docs/Initial_deployment.md

```diff
@@ -0,0 +1,153 @@
+
+During a one hour observation period, CyVerse had between 70 and 130 concurrent DB connections (90 on average). The DBMS was performing 140 IOPS on average with bursts of up to 3000.
+
+The storage will need to be able to hold the entire database as well write-ahead logs and DBMS bookkeeping information. The database files tend to grow larger over time due to rows in the middle of the file becoming invalidated due to deletes and updates. (Eventually these rows will be overwritten.) This growth will need to be accounted for when estimating how much storage will be needed.
+
+CyVerse's database is approximately 900 GiB in size, but PostgreSQL is using 1400 GiB of storage.
+
+#### Network
+
+Since the iRODS catalog provider will be making lots a requests to the DBMS with nearly all of these requests and their responses only requiring a small amount of data being transferred, the latency of the network connection between the catalog provider and the DBMS should be as small as reasonably possible.
+
+Network throughput is not that important. Over a one day period, CyVerse observed their catalog database had an average throughput of 4 MiB/s with a peak of 11 MiB/s.
+
+Institutional firewall appliances are known for increasing network jitter. They also have a habit of severing connections that are idle for too long. Some iRODS queries can take a few minutes to execute on a large database. To reduce the likelihood of broken database connections causing problems for users, it is recommended to not place the DBMS and the iRODS catalog provider on opposite sides of an institutional firewall.
+
+## Determining iRODS configuration
+
+This section gives advice on how to choosing and configuring hosts for iRODS and its catalog DBMS.
+
+### Deployment topology
+
+iRODS support a variety of deployment topologies. When determining which deployment topology to use, the system architect needs to decide whether or not to host the DBMS separately and whether or not to have separate resource servers among other things.
+
+#### Deciding whether to colocate catalog provider and DBMS
+
+In general, it is better to have the catalog provider on a separate server that the DBMS, but there are cases where it makes sense to collocate them as long as the DBMS is on a single host and is dedicated to iRODS.
+
+1. The iRODS zone will be small and lightly used. In other words, the zone won't have a lot of data objects and there won't be more than a few concurrent connections.
+1. The catalog provider will not be storing data object replicas locally, and the system is expected to be moderately used -- maybe less than 10 concurrent connections.
```

Comment:
number needs incrementing, I think (unless Markdown does that automatically)?

Also possible use case is for a test system.

Reply:
Markdown handles incrementing automatically. 

I added the test system use case. I'm not happy with how terse my sentence is, but I couldn't think of a good way to extend it. (Maybe my coffee hasn't kicked in.) Please feel free to expound on it.l